draft-irtf-cfrg-opaque.md

```diff
@@ -1060,35 +1063,50 @@ ServerInit(server_identity, server_private_key, server_public_key,
            record, credential_identifier, oprf_seed, ke1, client_identity)
 
 Input:
-- server_identity, the optional encoded server identity, which is set to
-  server_public_key if nil.
-- server_private_key, the server's private key.
-- server_public_key, the server's public key.
+- server_identity, the optional server identity.
+- server_private_key, the server's private key for the AKE protocol.
+- server_public_key, the server's public key for the AKE protocol.
 - record, the client's RegistrationRecord structure.
 - credential_identifier, an identifier that uniquely represents the credential.
-- oprf_seed, the server-side seed of Nh bytes used to generate an oprf_key.
+- oprf_seed, the server's OPRF seed.
 - ke1, a KE1 message structure.
-- client_identity, the encoded client identity.
+- client_identity, the optional client identity.
 
 Output:
 - ke2, a KE2 structure.
```

Comment:
Should this output type change?

Reply:
not sure, change to what type? (didn't get your question, could you please elaborate)

Reply:
Well, the function returns something of type `Response`, not `KE2`.

Reply:
No. It returns the result of calling `Response` function. (https://github.com/cfrg/draft-irtf-cfrg-opaque/blob/515717e8b6f19d427d6190f99450a850cfb709aa/draft-irtf-cfrg-opaque.md?plain=1#L1496)

Reply:
Oh, right! Sheesh. We need to really fix the names in this spec. 🤦